fs/du_unix.go

```diff
@@ -30,7 +30,7 @@ func diskUsage(roots ...string) (Usage, error) {
 
 			stat := fi.Sys().(*syscall.Stat_t)
 
-			inoKey := inode{dev: uint64(stat.Dev), ino: uint64(stat.Ino)}
+			inoKey := inode{dev: uint64(stat.Dev), ino: uint64(stat.Ino)} // nolint: unconvert
```

Comment:
Bug?

Reply:
Same here, it's uint64 for 64 bit platforms, but not on 32bit platforms.